README.md

```diff
@@ -19,7 +19,8 @@ steps:
 - uses: actions/checkout@latest
```

Comment:
Not really related to this PR, but could you change it from `actions/checkout@latest` to `actions/checkout@v1`. The build fails if you use `latest`. Want to have the YAML working correctly if people copy and paste it.

Reply:
Disregard. There was a another PR that fixed this. Some merge conflicts though that will have to be resolved.